httpserver/config.go

```diff
@@ -44,6 +45,7 @@ const (
 	defaultServerTimeoutsIdle       = time.Minute
 	defaultServerTimeoutsShutdown   = time.Second * 5
 	defaultSlowRequestThreshold     = time.Second
+	defaultTimeSlotsThreshold       = time.Second
```

Comment:
Suggest removing `defaultTimeSlotsThreshold` at all. In that case, `TimeSlotsThreshold`  will be the same as `SlowRequestThreshold` by default. Please do the same change for the grpc interceptor.